src/sanghwa/memberSet.java

```diff
@@ -0,0 +1,96 @@
+package sanghwa;
+
+import jun.Member;
+
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class memberSet {
```

Comment:
클래스 이름은 **반드시!!!!** 파스칼 케이스로 작성되어야 해요!

카멜 케이스, 파스칼 케이스에 대해서 찾아보세요!
만약 클래스 이름을 카멜 케이스로 작성되었을때 발생하는 문제에 대해서도 찾아보시고 공부해보세요!

```suggestion
public class MemberSet {
```